### DISCUSSION TITLE
example for 2040 hosting arduino serial

### DISCUSSION THREAD

**Discussion by beckmx** [hathach/tinyusb#1792] [Q&A]
hello guys, I was wondering if someone has an example of making the rpi2040 connect via usb host an old arduino uno board, those boards that do not use the ch3xx chipsets

**Comment by hathach** [comment]
cdc host support is just added recently with TinyUSB Arduino. It works with other tinyusb running cdc, you can give it a try https://github.com/adafruit/Adafruit_TinyUSB_Arduino/tree/master/examples/DualRole/serial_host_bridge

**Comment by beckmx** [comment]
I am going to test
